packages/syft/src/syft/custom_worker/config.py

```diff
@@ -140,3 +143,6 @@ def test_image_build(self, tag: str, **kwargs) -> Tuple[Image, SyftSuccess]:
             return SyftSuccess(message=parse_output(logs))
         except Exception as e:
             return SyftError(message=f"Failed to build image !! Error: {str(e)}.")
+
+    def set_description(self, new_description: str) -> None:
```

Comment:
Maybe we rename the variable from `new_description` to `description_text`

Reply:
Done.